app/controllers/application_controller/filter.rb

```diff
@@ -1,10 +1,13 @@
 # Filter/search/expression methods included in application.rb
+require_relative '../../helpers/miq_expression/filter_subst'
```

Comment:
@Fryguy 
I have a separate WIP PR for Automate Expression Method https://github.com/ManageIQ/manageiq/pull/6655
where I am going to use these functions.

The 4 methods that I included in the helper file were extracted from the app/controllers/application_controller/filter.rb where it was only available to the UI and the controllers.

I would need these functions to do substitution from the automate expression methods. These instance methods which are in a module can be included by the app/controllers/application_controller/filter.rb as well as the lib/miq_automation_engine/engine/miq_ae_expression_method.rb in my WIP PR.